packages/backend/test/core/transaction-count/RpcTransactionUpdater.test.ts

```diff
@@ -142,6 +142,70 @@ describe(RpcTransactionUpdater.name, () => {
         ],
       ])
     })
+
+    it('subtracts Nitro system transactions', async () => {
+      const TIME_0 = new UnixTime(0)
+      const ethereumClient = mock<EthereumClient>({
+        getBlock: async () =>
+          fakeBlock({
+            number: 1,
```

Comment:
to make this mock correct here we should return actual block number that was asked.